lib/cockpit-components-dialog.jsx

```diff
@@ -226,26 +226,32 @@ var Dialog = React.createClass({
 
 /* Create and show a dialog
  * For this, create a containing DOM node at the body level
+ * returns a function that the caller can use to update the body (footer will remain unchanged)
  */
-var show_modal_dialog = function(dialog_props, footer_props) {
+var show_modal_dialog = function(dialogProps, footerProps) {
```

Comment:
The whole sale renaming of variables makes it hard to review, but will do my best.